pro/src/routes/OfferEducationalEdition/utils/computeInitialValuesFromOffer.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  DEFAULT_EAC_FORM_VALUES,
+  IOfferEducationalFormValues,
+  PARTICIPANTS,
+} from 'core/OfferEducational'
+import { Offer } from 'custom_types/offer'
+
+const computeDurationString = (durationMinutes: number | undefined) => {
+  if (!durationMinutes) return '00:00'
```

Comment:
pourquoi pas : 
```suggestion
  if (!durationMinutes) return DEFAULT_EAC_FORM_VALUES.duration
```

?